packages/account/src/providers/transaction-summary/get-transaction-summary.ts

```diff
@@ -142,7 +142,17 @@ export async function getTransactionsSummaries(
 ): Promise<GetTransactionsSummariesReturns> {
   const { filters, provider, abiMap } = params;
 
-  const { transactionsByOwner } = await provider.operations.getTransactionsByOwner(filters);
+  const { owner, ...inputArgs } = filters;
+
+  const validPaginationParams = provider.validatePaginationArgs({
+    inputArgs,
+    paginationLimit: TRANSACTIONS_PAGE_SIZE_LIMIT,
+  });
+
+  const { transactionsByOwner } = await provider.operations.getTransactionsByOwner({
+    owner,
+    ...validPaginationParams,
```

Comment:
```suggestion
    ...validPaginationParams,
    owner,
```

Just to make sure it's not overwritten